### PR TITLE
ER-526: Added check of reviews

### PR DESCRIPTION
**Pull request by rimi-itk**
About 1 in 5 (at last count) of the reviews from litteratursiden.dk do not exist anymore. To remedy this, we add a function to check the http status code of each review and only use reviews that actually exist (when last checked).

```
mysql> select http_code, count(*) from reol_review_reviews group by http_code;
+-----------+----------+
| http_code | count(*) |
+-----------+----------+
|       200 |     6681 |
|       404 |     1247 |
+-----------+----------+
3 rows in set (0.00 sec)
```

### CODE REVIEW

File: sites/all/modules/reol_review/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Reol review
+
+## Check availability of reviews
+
+Run the `'reol_review_check` to check availability of reviews, i.e. check the
+http code when `GET`'ing the actual review.
+
+```sh
+drush php-eval 'reol_review_check()'
```
Comment (cableman) [inline sites/all/modules/reol_review/README.md]: Why not make an drush file an make it a real drush command. 20 linjes of code?

File: sites/all/modules/reol_review/reol_review.install [hunk 2]
```diff
@@ -143,6 +143,31 @@ function _reol_review_schema_2() {
   return $schema;
 }
 
+/**
+ * Fourth schema version.
+ */
+function _reol_review_schema_3() {
```
Comment (cableman) [inline sites/all/modules/reol_review/reol_review.install]: This scheme thing... is a hack. Also remeber to update `reol_review_schema()` so the real scheme match.

File: sites/all/modules/reol_review/README.md (hunk 1 above)
Comment (rimi-itk) [reply sites/all/modules/reol_review/README.md]: Stuff moved to `drush` command.

File: sites/all/modules/reol_review/reol_review.install (hunk 2 above)
Comment (rimi-itk) [reply sites/all/modules/reol_review/reol_review.install]: Fixed.

**Pull request merged**